docs/src/theme/DocSidebar/Desktop/Content/styles.module.css

```
@@ -41,33 +39,33 @@
     animation: contextual-pane-enter 150ms cubic-bezier(0.175, 0.885, 0.32, 1.1);
   }
 
-  .bottomFade {
-    position: absolute;
-    right: 0.75rem;
+  .versionControl {
```

Stylelint (17.14.0)

[error] Expected class selector ".versionControl" to be kebab-case (selector-class-pattern)

(selector-class-pattern)